.gitignore

```diff
@@ -13,3 +13,4 @@ BROWSE
 *.pyc
 SOURCE_VERSION
 .cache
+.project
```

Comment:
This probably shouldn't be here.

Reply:
I am using CLion for dev and it keeps it's project config in .project. Will remove from the PR.